data_processing/scanManager/extractRadiomics.py

```diff
@@ -50,9 +50,9 @@ def add_container_data(container_id, n_meta):
     conn = Neo4jConnection(uri=os.environ["GRAPH_URI"], user="neo4j", pwd="password")
 
     res = conn.query(f""" 
-        MATCH (sc:scan) WHERE id(sc)={container_id}
+        MATCH (object) WHERE id(object)={container_id}
```

Comment:
Would like to come up with a better query variable name here, "container"?

Reply:
container sounds good!